pkgs/tools/package-management/harmonia/default.nix

```diff
@@ -35,6 +36,7 @@ rustPlatform.buildRustPackage rec {
     updateScript = nix-update-script {
       extraArgs = [ "--version-regex" "harmonia-v(.*)" ];
     };
+    tests = { inherit (nixosTests) harmonia; };
   };
 
   meta = with lib; {
```

Comment:
I just noticed that the meta section needs an update

Reply:
done.